scripts/tp_dump.py

```diff
@@ -100,15 +100,20 @@ def plot_pdf_adc_integral_vs_peak(tp_data: np.ndarray, pdf: PdfPages, verbosity:
         )
         print("Total number of TPs:", len(tp_data['adc_peak']))
     high_integral_locs = np.where(tp_data['adc_integral'] == np.power(2, 15)-1)
+    unity = (np.min(tp_data['adc_peak']), np.max(tp_data['adc_peak']))
```

Comment:
This works because physically `adc_peak` should be always smaller than `adc_integral`, but effectively that was not always the case (because bugs, overflows with avx etc). 

Perhaps a check if `np.max(tp_data['adc_peak'])` is definitely smaller than `np.max(tp_data['adc_integral'])` would be useful here, and if not, printout warning & use `np.max(tp_data['adc_integral'])` instead for the line of unity.

Probably not needed for this PR though, just something to bear in mind...